types/deep-equal/index.d.ts

```diff
@@ -12,4 +12,4 @@ declare function deepEqual(
     expected: any,
     opts?: DeepEqualOptions): boolean;
 
-export = deepEqual;
+export default deepEqual;
```

Comment:
[`deep-equal` doesn't have a default export](https://github.com/substack/node-deep-equal/blob/master/index.js#L5).

```bash
$ node -p 'require("deep-equal")'
[Function]
$ node -p 'require("deep-equal").default'
undefined
```

[See the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default) for instructions on how to configure your build environment to fix this error (assuming the module bundler you're using supports it)

Reply:
good point! I didn't know about --esModuleInterop, that would have fixed it. But I'm using `_.isEqual` now, works without changing anything. So delete this PR?